### PR TITLE
Fix suppression fixer to correctly detect the enclosing member when g…

### LINKED ISSUE #6379: Global suppression for diagnostic on trivia of an enum member doesn't work
1. Create a C# project with below code:

  ````c#
  public enum E
  {
    M1,     
    M2
  }
  ````

2. Install codecracker analyzer from nuget.org
3. Select diagnostic CC0065 for trailing whitespace after field M1 in error list and invoke command to suppress in suppressions file
4. The below suppression gets added to the global suppression file, this is as expected:

  ```c#
[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Style", "CC0065:Remove trailing whitespace", Justification = "<Pending>", Scope = "member", Target = "~F:E.M1")]
  ```

**Expected:** CC0065 diagnostic goes away as it got suppressed.
**Actual:** Diagnostic still shows up as active.

**Reason:** It seems that the global suppression is being applied to the span of member M1, not its fullspan which includes the trivia - this is causing the diagnostic to still be unsuppressed.

### PR DESCRIPTION
**Pull request by mavasani**
…enerating a global suppression for a diagnostic in trivia.

Fixes #6379

### CODE REVIEW

File: src/Features/Core/Portable/CodeFixes/Suppression/AbstractSuppressionCodeFixProvider.cs [hunk 1]
```diff
@@ -195,7 +195,7 @@ internal class SuppressionTargetInfo
             var syntaxFacts = document.GetLanguageService<ISyntaxFactsService>();
 
             ISymbol targetSymbol = null;
-            var targetMemberNode = syntaxFacts.GetContainingMemberDeclaration(root, startToken.SpanStart);
+            var targetMemberNode = syntaxFacts.GetContainingMemberDeclaration(root, nodeWithTokens.SpanStart);
```
Comment (jaredpar) [inline src/Features/Core/Portable/CodeFixes/Suppression/AbstractSuppressionCodeFixProvider.cs]: For my education: in what case would these two values be different?  I would assume the start token of any node had the same `SpanStart` as that node.

Comment (mavasani) [inline src/Features/Core/Portable/CodeFixes/Suppression/AbstractSuppressionCodeFixProvider.cs]: `startToken` need not be the first token for `nodeWithTokens`, which is a first common node containing start and end tokens. For example, for the test case for this bug:

```c#
enum E
{
  F1, // some comment
  F2
}
```

`startToken` = `F1` and `endToken` = `,` token and closest node containing both tokens is the enum declaration which has a different span start.

**Comment by mavasani** [comment]
@heejaechang @srivatsn @basoundr

**Comment by heejaechang** [comment]
:+1:

**Pull request merged**